packages/terra-application-layout/src/header/ApplicationHeader.module.scss

```diff
@@ -2,8 +2,10 @@
   .application-header {
     background-color: var(--terra-application-header-background-color, #0079be);
     border: var(--terra-application-header-border, 0.07143rem solid rgba(255, 255, 255, 0.17));
+    box-shadow: var(--terra-application-layout-navbar-box-shadow);
```

Comment:
This variable is pulled from the [OCS theme branch](https://github.com/cerner/terra-framework/blob/3152c95dd5c9c39f028a92d2d2e884549d282331/packages/terra-application-layout/src/header/ApplicationHeader.module.scss#L6) along with [the z-index addition](https://github.com/cerner/terra-framework/blob/3152c95dd5c9c39f028a92d2d2e884549d282331/packages/terra-application-layout/src/header/ApplicationHeader.module.scss#L11).